main.py

```diff
@@ -1,6 +1,6 @@
 import argparse
 import sys
-from stix_shifter import stix_shifter
+from stix_shifter.stix_translation import stix_translation
```

Comment:
Could we move the transmit method into stix_transmission.py? You could then add 
`from stix_shifter.stix_transmission import stix_transmission` here. Right now when I call transmit from the CLI, I get an error saying no module named 'stix_transmission'.